docs/tutorial/part05.rst

```diff
@@ -2,5 +2,126 @@
 How to create the API.
 ======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Why the API structure
+=====================
+
+We decided to use APIs to fetch the data to make queries more effective, as sometimes queries get
+too big. We decided it was better to fetch the data in little chunks rather than one big request.
+The API structure allows us to do so and to have a single page application. We will be returning
+JSON to the calls since it is easily usable on the frotend side.
+
+Explain the library

```

Comment:
'Explain the library' sounds weird (boring to be honest). Instead use something like 'REST API' or 'Implementing REST framework for API' it's more attractive and useful for somebody that is only looking for this.